Flaky test: test_scheduler_cpu_preemptive/test.py::test_downscaling[cpu-slot-preemption-timeout-1ms]

_Important: This issue was automatically generated and is used by CI for matching failures. DO NOT modify the body content. DO NOT remove labels._

Test name: test_scheduler_cpu_preemptive/test.py::test_downscaling[cpu-slot-preemption-timeout-1ms]
Failure reason: Timeout
CI report: [Integration tests (amd_tsan, 2/6)](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=92847&sha=c023a745e3bed7155897cd3dfeca63f4e427f611&name_0=PR&name_1=Integration%20tests%20%28amd_tsan%2C%202%2F6%29)

CIDB statistics: [cidb](https://play.clickhouse.com/play?user=play&run=1#V0lUSAogICAgOTAgQVMgaW50ZXJ2YWxfZGF5cwpTRUxFQ1QKICAgIHRvU3RhcnRPZkRheShjaGVja19zdGFydF90aW1lKSBBUyBkYXksCiAgICBjb3VudCgpIEFTIGZhaWx1cmVzLAogICAgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlcikgQVMgcHJzLAogICAgYW55KHJlcG9ydF91cmwpIEFTIHJlcG9ydF91cmwKRlJPTSBjaGVja3MKV0hFUkUgKG5vdygpIC0gdG9JbnRlcnZhbERheShpbnRlcnZhbF9kYXlzKSkgPD0gY2hlY2tfc3RhcnRfdGltZQogICAgQU5EIHRlc3RfbmFtZSA9ICd0ZXN0X3NjaGVkdWxlcl9jcHVfcHJlZW1wdGl2ZS90ZXN0LnB5Ojp0ZXN0X2Rvd25zY2FsaW5nW2NwdS1zbG90LXByZWVtcHRpb24tdGltZW91dC0xbXNdJwogICAgLS0gQU5EIGNoZWNrX25hbWUgPSAnSW50ZWdyYXRpb24gdGVzdHMgKGFtZF90c2FuLCAyLzYpJwogICAgQU5EIHRlc3Rfc3RhdHVzIElOICgnRkFJTCcsICdFUlJPUicpCiAgICBBTkQgKChwdWxsX3JlcXVlc3RfbnVtYmVyID0gMCBBTkQgaGVhZF9yZWYgPSAncGFydGlhbC1tYXRjaC1qc29uJykgT1IgKHB1bGxfcmVxdWVzdF9udW1iZXIgIT0gMCBBTkQgYmFzZV9yZWYgPSAnbWFzdGVyJykpCkdST1VQIEJZIGRheQpPUkRFUiBCWSBkYXkgREVTQwo=)

Test output:
```
File: test_scheduler_cpu_preemptive/test.py:416 - in test_downscaling
    development.stop(tid)
File: test_scheduler_cpu_preemptive/test.py:376 - in stop
    thread.join()
File: /usr/lib/python3.10/threading.py:1096 - in join
    self._wait_for_tstate_lock()
File: /usr/lib/python3.10/threading.py:1116 - in _wait_for_tstate_lock
    if lock.acquire(block, timeout):
E   Failed: Timeout >900.0s
```

--- a/tests/integration/test_scheduler_cpu_preemptive/test.py
+++ b/tests/integration/test_scheduler_cpu_preemptive/test.py
@@ -406,7 +406,7 @@ def test_downscaling(with_custom_config):
     node.query(
         f"""
         create resource cpu (master thread, worker thread);
-        create workload all settings max_concurrent_threads=8;
+        create workload all settings max_concurrent_threads=2;
         create workload development in all;
     """
     )
@@ -416,9 +416,9 @@ def test_downscaling(with_custom_config):
     active_ids: list[int] = []
     try:
         for _ in range(2):
-            # Gradually increase concurrency to 16
-            for _ in range(16):
-                tid = development.start(billions=1, max_threads=8)
+            # Gradually increase concurrency to 4
+            for _ in range(4):
+                tid = development.start(billions=1, max_threads=2)
                 active_ids.append(tid)
                 time.sleep(0.05)
             # Gradually decrease back to 0

